package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.7.2",
   "author": "Matthias Kretschmann <m@kretschmann.io>",
   "scripts": {
+    "prepare": "npm run build",
```

Comment:
This script should not be here, at least I can't see what the advantage of adding this script here is. Would you mind removing it?